src/number_theory/number_field/embeddings.lean

```diff
@@ -51,38 +51,25 @@ open set polynomial
 /-- Let `A` an algebraically closed field and let `x ∈ K`, with `K` a number field. For `F`,
 subfield of `K`, the images of `x` by the `F`-algebra morphisms from `K` to `A` are exactly
 the roots in `A` of the minimal polynomial of `x` over `F`. -/
-lemma range_eq_roots (F K A : Type*) [field F] [number_field F] [field K] [number_field K]
-  [field A] [is_alg_closed A] [algebra F K] [algebra F A] (x : K) :
+-- ! should be moved to another namespace
+lemma range_eq_roots (F K A : Type*) [field F] [field K] [field A] [is_alg_closed A]
+  [algebra F K] (hK : algebra.is_algebraic F K) [algebra F A] (x : K) :
   range (λ ψ : K →ₐ[F] A, ψ x) = (minpoly F x).root_set A :=
```

Comment:
RFC: where should this be moved to?

Reply:
When don't you leave it here and change the namespace to `is_algebraic`? Or you could move to `is_algebraic` but it does fit here naturally. I guess it is acceptable to have results that are more general than `number_field` in the `number_field` directory.

Reply:
Thanks for the great suggestions! I've made F, K implicit and renamed it `algebra.is_algebraic.range_eval_eq_root_set_minpoly`, but I decided to move it to *field_theory/is_alg_closed/basic*.

The two `haveI : is_scalar_tower` lines are a bit slow, making the whole proof taking >5 seconds to check, and I'm not sure why.